src/adapter/src/catalog.rs

```diff
@@ -428,6 +428,10 @@ impl CatalogState {
         &self.roles_by_id[id]
```

Comment:
smallest nit: can you change this to `self.roles_by_id.get(id).expect("catalog out of sync")`, that will provide a better error message than the bracket syntax

Reply:
Done